cfgov/jinja2/v1/_includes/organisms/ask-search.html

```diff
@@ -57,7 +57,22 @@
                                 input-contains-label
                                 {% if autocomplete %}m-autocomplete{% endif %}"
                          data-language="{{ language }}">
-                        <label for="query" class="input-contains-label_before input-contains-label_before__search"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 880.6 1200" class="cf-icon-svg"><path d="M860.1 879.2L645.7 664.8c90.8-136.8 76-323-44.6-443.5-137.5-137.5-360.5-137.5-498 0s-137.5 360.5 0 498c118.5 118.4 303.9 137 443.5 44.6L761 978.3c27.3 27.3 71.7 27.3 99 0s27.4-71.8.1-99.1zm-508-116.9C191.4 762 61.3 631.5 61.6 470.8c.3-160.7 130.8-290.8 291.5-290.5s290.8 130.8 290.5 291.5c-.2 118.2-71.9 224.6-181.5 269.1-34.9 14.2-72.3 21.5-110 21.4z"></path></svg></label>
+                        {% if show_label %}
+                        <span
```

Comment:
Hmm I wonder if we want to have open tags like this that visually look like a mistake of omitting the `>` on first glance. Maybe we should have something like this:

```
{% if show_label %}
  {% set html_tag = 'span' %}
{% else %}
  {% set html_tag = 'label for="o-search-bar_query"` %}
{% endif %}
<{{ html_tag }} class="input-contains-label_before input-contains-label_before__search">
```

Reply:
@anselmbradford, good call. I took a similar approach in dc6d8f1 but used a couple of inline if expressions to (a) try and keep it a little more readable and (b) be able to reuse the same variable in the closing tag. How's that looking?